applications/luci-app-dawn/Makefile

```diff
@@ -0,0 +1,9 @@
+include $(TOPDIR)/rules.mk
+
+LUCI_TITLE:=LuCI support for DAWN
+LUCI_DEPENDS:=+luci +dawn +luci-lib-json +luci-lib-httpclient +luci-compat
```

Comment:
Dependency on the "luci" meta-package sounds wrong.

Reply:
Oh, okay, I will delete the `+luci`. I had this in my own repository and I think this was needed. I will delete that.